### DISCUSSION TITLE
Wanting to make a Spotify Client for my Nokia 130?

### DISCUSSION THREAD

**Discussion by lafeo** [raspiduino/mre-sdk#9] [Q&A]
I am very, very new to this side of Software Dev, clearly. But I just need to figure out a way to use Spotify's extensive API's to somehow, at least, stream Music and Browse playlists on Nokia S30+ phone?

**Comment by raspiduino** [comment]
> to use Spotify's extensive API's

Does it support http-only traffic? If not, then you would better do a proxy stuff (eg running a program on another machine that calls to Spotify APIs and then forward the result in your format to Nokia client).

**Comment by XimikBoda** [comment]
Https can be implemented on Nokia (the prototype is ready, but it still needs to be tested), the next problem is that the official Spotify API does not provide direct access to music (only the first 30 seconds).

P.C. This is possible for premium profile.
